packages/studio-web/src/app/app.component.ts

```diff
@@ -258,7 +258,8 @@ export class AppComponent implements OnDestroy, OnInit {
 
   openPrivacyDialog(): void {
     this.dialog.open(PrivacyDialog, {
-      width: "250px",
+      width: "50vw",
+      maxHeight: "90vh",
```

Comment:
Just wondering, what's a vw and what's a vh?

Reply:
view width and view height. 1vX is 1% of the viewport width or viewport height. so it's instead of pixels (which on some screens don't nicely correspond with viewport size...looking at youuuu iPhones 😄 ). I should be more consistent in my use of it maybe, but I usually use it just in these cases for large containers and things like that